ui/src/services/loki/api.js

```diff
@@ -0,0 +1,39 @@
+//@flow
+import ApiClient from '../ApiClient';
+
+let lokiApiClient: ?ApiClient = null;
+
+export function initialize(apiUrl: string) {
+  lokiApiClient = new ApiClient({ apiUrl });
+}
+
+export function getAlertsLoki(start: string, end: string) {
+  if (!lokiApiClient) {
+    throw new Error('lokiApiClient should be defined');
+  }
+  // TODO: Hardcode for now
+  const query =
+    "{namespace='metalk8s-monitoring',container='alertmanager-logger'}";
```

Comment:
According to @alexandre-allard-scality 
We will need to specify the container name. Otherwise, we'll end up with the logs of everything.

But yes, you are right, we don't need to specify namespace here.

Reply:
Let's keep the namespace, just in case, there's no harm in being specific here. We're reading logs from a specific pod.

Reply:
Hi, @gdemonet We were thinking about exporting the `getAlertsLoki` function through module federation so that other UIs could benefit from the alert fetching function. (@JBWatenbergScality can you confirm this?)

And then apply the filter {namespace: 'metalk8s-monitoring'}? 
WDYT?

Reply:
I confirm the idea would be to use the same function in others scopes so if this namespace filter in query parameters refrain us to do so I think we should get rid of it.